packages/features/flags/components/FlagAdminList.tsx

```diff
@@ -32,7 +32,7 @@ const FlagToggle = (props: { flag: Flag }) => {
     flag: { slug, enabled },
   } = props;
   const utils = trpc.useContext();
-  const mutation = trpc.viewer.features.toggle.useMutation({
+  const mutation = trpc.viewer.admin.toggleFeatureFlag.useMutation({
```

Comment:
Moved to admin so we can set the cache control on the createNextAPIHandler to true as a result of isPublic=true